lib/kennel/models/slo.rb

```diff
@@ -35,7 +35,9 @@ def build_json
           type: type
         )
 
-        if v = query
+        if type == 'time_slice'
+          data[:sliSpecification] = :sli_specification
```

Comment:
this needs to be the actual call ?
```suggestion
          data[:sliSpecification] = sli_specification
```